modules/react/icon/lib/SystemIcon.tsx

```diff
@@ -46,7 +46,7 @@ export interface SystemIconStyles {
 
 export interface SystemIconProps
   extends SystemIconStyles,
-    Omit<IconProps, 'src' | 'color' | 'type' | 'background'> {
+    Omit<IconProps, 'src' | 'color' | 'fill' | 'type' | 'background'> {
```

Comment:
Omitting this so the `fill` style prop doesn't conflict with the `fill` prop in `SystemIconStyles`.